src/options.d.ts

```diff
@@ -1,6 +1,8 @@
 export interface FaviconWebpackPlugionOptions {
   /** our source logo - can be png or svg (required) */
   logo: string | string[],
+  /** our maskable source logo - can be png or svg (optional) */
+  logoMaskable: string | string[],
```

Comment:
As soon as it is optional, it should be reflected in the type too.

Reply:
done 👍